api/src/pcapi/routes/serialization/collective_offers_serialize.py

```diff
@@ -623,6 +629,23 @@ def validate_booking_emails(cls, booking_emails: list[str]) -> list[str]:
             raise ValueError("Un email doit etre renseigné.")
         return booking_emails
 
+    @root_validator
+    def validate_location_fields(cls, values: dict) -> dict:
+        has_offer_venue = values.get("offer_venue") is not None
+        has_oa = (
+            (values.get("location_type") is not None)
+            or (values.get("location_comment") is not None)
+            or (values.get("address") is not None)
+        )
```

Comment:
`location_type` est obligatoire dans le cas ou on envoie oa. On devrait plutot avoir un truc du genre :


```suggestion
            values.get("location_type") is not None
        )
```

Reply:
Oui, pour le POST j'imagine qu'on peut se dire ça vu qu'on reçoit tous les champs

Reply:
-> champ location avec sous-champs + locationType obligatoire